Remove "Teams-only apps" section of TeamsJS overview (#10588)
MicrosoftDocs/msteams-docs: msteams-platform/tabs/how-to/using-teams-client-library.md

You can check for host support of a given capability at runtime by calling the `isSupported()` function on that capability (namespace). It returns `true` if it's supported and `false` if not, and you can adjust app behavior as appropriate. This allows your app to light up UI and functionality in hosts that support it, while continuing to run for hosts that don't.

> The host name where your app operates is exposed as a [HostName](/javascript/api/%40microsoft/teams-js/hostname) enum value off the Context interface (`app.Context.app.host.name`). You can query this at runtime by invoking `getContext`. For the previous Teams client, this value might return as *unknown* or *undefined*. In such cases, map these values to old Teams.

Reviewer: ```suggestion
The host name where your app operates is displayed as a [HostName](/javascript/api/%40microsoft/teams-js/hostname) enum value of the `Context` interface (`app.Context.app.host.name`). You can query this at runtime by invoking `getContext`. For the classic Teams client, this value might return as *unknown* or *undefined*. In this case, map these values to classic Teams.
```

Reviewer: @erikadoyle For expose, we've updated as per MSTP guidelines https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/expose
